chrome/content/zotero/elements/editableText.js

```diff
@@ -218,6 +218,11 @@
 				}
 			}
 			this._input.readOnly = this.readOnly;
+			if (this.readOnly) {
+				// Voiceover does not announce text as readonly. aria-readonly gets ignored as well.
+				// aria-disabled is the only found working property - it announced the text as "dimmed"
+				this._input.setAttribute("aria-disabled", true);
```

Comment:
Good catch, thank you!

As I was testing it, I still couldn't reproduce this with NVDA/JAWS. And if it is not an issue on windows, we don't want to add `aria-disabled` because then textfields are announced as both "readonly" and "unavailable", which sounds confusing. So I thought we can do this to not affect windows for now:
```
if (Zotero.isMac) {
   // setAttribute to make sure it's always synched with this.readOnly
    this._input.setAttribute("aria-disabled", this.readOnly);
}
```
And then maybe we'll check with VPAT reviewers to confirm if/when `readonly` is not announced on windows